parsely/src/main/java/com/parsely/parselyandroid/Log.kt

```diff
@@ -0,0 +1,34 @@
+package com.parsely.parselyandroid
+
+import android.util.Log as AndroidLog
+
+internal object AndroidLogWrapper : Log {
```

Comment:
Minor (🔍): Why the `Android` prefix, wouldn't `LogWrapper` be enough? 🤔

PS: Nothing serious, I just wonder. 😊

Reply:
It's a wrapper around Android's `Log` so I went with `AndroidLogWrapper`.

Reply:
Just because we are anyway in the context of Android, to me this was a given, thus redundant, words! 😅